web-app/js/portal/ui/openlayers/layer/NcWms.js

```diff
@@ -278,7 +279,7 @@ OpenLayers.Layer.NcWms = OpenLayers.Class(OpenLayers.Layer.WMS, {
     },
 
     _getTimeParameter: function(dateTime) {
-        return dateTime.clone().utc().format('YYYY-MM-DDTHH:mm:ss.SSS');
+        return dateTime.clone().utc().format('YYYY-MM-DDTHH:mm:ss.SSS') + 'Z';
```

Comment:
`console.log(time.utc().format('YYYY-MM-DDTHH:mm:ss.SSSZ'));` results in `2011-07-08T03:32:45.000+00:00`

So not good. Leave as is.